android/assets/jsons/translations/template.properties

```
@@ -1600,6 +1600,13 @@ Toggle UI (World Screen only) =
 Overrides yields from underlying terrain = 
 No yields = 
 Mod: [modname] = 
+Search text: = 
+That's an invalid regular expression. = 
```

```suggestion
Invalid regular expression = 
```
Just as informative, and shorter